printer_mods/Takuya/USBCameraMount/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+This my my contribution to the awesome Voron Community!
+
+Its a camera mount for a USB Camera.
+
+![Screenshot_4](https://user-images.githubusercontent.com/54855101/126408342-3fb0ea8e-ef64-45e4-821c-6806f4884df7.jpg)
```

Comment:
Please upload this image to the repository and reference it from there.  For example, you could upload it as `printer_mods/Takuya/USBCameraMount/screenshot4.jpg` and then reference it in this readme as

    ![Screenshot_4](screenshot4.jpg)

Reply:
Thanks,
done